src/platform_impl/macos/window.rs

```diff
@@ -359,11 +338,11 @@ impl UnownedWindow {
 
         let pool = unsafe { NSAutoreleasePool::new(nil) };
 
-        create_app(pl_attribs.activation_policy).ok_or_else(|| {
+        let ns_app = unsafe { NSApp() };
+        if ns_app == nil {
             unsafe { pool.drain() };
-            os_error!(OsError::CreationError("Couldn't create `NSApplication`"))
-        })?;
-
+            return Err(os_error!(OsError::CreationError("`NSApp()` returned nil")));
+        }
```

Comment:
I don't think there's a reason for calling `NSApp` here any longer?

Reply:
Thanks for pointing this out I absolutely did not realize that.